tslib/src/functions/school-pin-generator/school-pin-expiry-generator.ts

```diff
@@ -0,0 +1,40 @@
+import moment from 'moment'
```

Comment:
moment-timezone package itself loads moment as a dependency, you are loading two largely identical versions of moment otherwise.